pkgs/development/python-modules/django_silk/default.nix

```diff
@@ -23,15 +25,23 @@
 
 buildPythonPackage rec {
   pname = "django-silk";
-  version = "4.0.1";
+  version = "4.1.0";
 
   # pypi tarball doesn't include test project
   src = fetchFromGitHub {
     owner = "jazzband";
     repo = "django-silk";
     rev = version;
-    sha256 = "0yy9rzxvwlp2xvnw76r9hnqajlp417snam92xpb6ay0hxwslwqyb";
+    sha256 = "1km3hmx1sir0c5gqr2p1h2938slhxp2hzf10cb80q98mas8spjkn";
   };
+
+  patches = lib.optional (pythonAtLeast "3.9") (fetchpatch {
```

Comment:
Second thought: Can we apply this patch on all versions?

Reply:
I considered that but thought it's nicer to actually be shipping literal 4.1.0 where we say we're giving the user "4.1.0" in cases that we easily can.